### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by luzpaz**
Fixes various typos

### CODE REVIEW

File: doc/examples/customers_import.csv [hunk 1]
```diff
@@ -8,7 +8,7 @@ id,company,name,addr1,addr2,addr3,addr4,phone,fax,email,notes,shipname,shipaddr1
 #company with the same ID will be UPDATED.  This may not be what you want!
 000099,Average Company,Accounts Dept,50 Poor Avenue,,,,,,,,,,,,,,,
 ,Academy,Academy,Some Street,,,,555-237-6959,,,,,,,,,,shipmail
-,company,name,addr1,addr2,addr3,addr4,phone,fax,emai,lnotes,shipname,shipaddr1,shipaddr2,shipaddr3,shipaddr4,shipphone,shipfax,shipemail
+,company,name,addr1,addr2,addr3,addr4,phone,fax,email,lnotes,shipname,shipaddr1,shipaddr2,shipaddr3,shipaddr4,shipphone,shipfax,shipemail
```
Comment (luzpaz) [inline doc/examples/customers_import.csv:11]: this isn't fixed. the next word `lnotes should be `notes`

File: libgnucash/app-utils/calculation/fin.c [hunk 2]
```diff
@@ -149,9 +149,9 @@
  *
  * 1)
  *                                                                 FV*
- *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
- * Period ���������������������������������������������������������
- *        �
+ *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
+ * Period ���������������������������������������������������������
+ *        �
```
Comment (luzpaz) [inline libgnucash/app-utils/calculation/fin.c:154]: Added by vscode. I will try to revert.

File: libgnucash/app-utils/calculation/fin.c [hunk 3]
```diff
@@ -149,9 +149,9 @@
  *
  * 1)
  *                                                                 FV*
- *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
- * Period ���������������������������������������������������������
- *        �
+ *          1   2   3   4   .   .   .   .   .   .   .   .   .   n �
+ * Period ���������������������������������������������������������
```
Comment (fellen) [inline libgnucash/app-utils/calculation/fin.c:153]: Are you using a different encoding than utf-8, perhaps Microsofts ancestor of utf-16?

Comment (luzpaz) [reply libgnucash/app-utils/calculation/fin.c]: i honestly don't know how to fix this encoding issue. I've googled around but can't seem to find the right search term. 

Anyway,  I've manually edited the changes in vim, saved, and then squashed; theoretically the changes should cancel each other out. But instead the diff shows I'm overwriting with my changes which are identical to the maint branch?? Weird. 

image

Sorry for the noise. Maybe a merge dev can swoop in and cherrypick the appropriate changes (without the encoding nonsense my IDE introduced in this commit?)

**Comment by jralls** [comment]
Not really odd, just ancient. It's the old IBM PC character set, [code page 437](https://en.wikipedia.org/wiki/Code_page_437); the more modern expression of the offending characters is the unicode [box-drawing block](https://en.wikipedia.org/wiki/Box-drawing_character).

The first clue is the line near the top of the file " * This version for use WITH ANSI.SYS display driver" which on its own should have caused @jdavisp3 to reject it. That statement is false, BTW, the file doesn't have any ANSI escapes. 

We can just lose the diagrams. We could lose most of the explanation, most people who'll use and one hopes anyone who'd work on the implementation of these functions understand the material pretty thoroughly already.

**Comment by fellen** [comment]
Just run 'iconv -f CP437 -t UTF-8 -o fin.c fin.c'
and remove the ANSI.SYS line.

**Comment by gjanssens** [comment]
That would convert a c source file to utf-8. While that's technically allowed I have read mixed opinions about that.
I personally am fine with that, just wanted to be sure we give it some thought beforehand.

**Comment by jralls** [comment]
Well, it at least makes the diagrams somewhat intelligible, but still pretty pointless to anyone even slightly familiar with the formulas.
```
*
 * 1)
 *                                                                 FV*
 *          1   2   3   4   .   .   .   .   .   .   .   .   .   n │
 * Period ┌───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
 *        │
 *
 *        PV
 *
 *     Appreciation
 *     Depreciation
 *     Compound Growth
 *     Savings Account
 *
 * ****************************************************************************
 *
 * 2)                                                               FV
 *     PV = 0
 *                                                                │
 * Period ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┘
 *        │ 1 │ 2 │ 3 │ 4 │ . │ . │ . │ . │ . │ . │ . │ . │ . │ n
 *
 *       PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT
 *
 *     Annuity (series of payments)
 *     Pension Fund
 *     Savings Plan
 *     Sinking Fund
 *
 * ****************************************************************************
 *
 * 3)
 *     PV
 *        │                                                      FV=0
 * Period └───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
 *          1 │ 2 │ 3 │ 4 │ . │ . │ . │ . │ . │ . │ . │ . │ . │ n │
 *
 *           PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT
 *
 *     Amortization
 *     Direct Reduction Loan
 *     Mortgage (fully amortized)
 *
 * ****************************************************************************
 *
 * 4)
 *                                                                 FV*
 *           PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT PMT  │ +
 *                                                    PMT
 *          1 │ 2 │ 3 │ 4 │ . │ . │ . │ . │ . │ . │ . │ . │ . │ n │
 * Period ┌───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
 *        │
 *
 *        PV
 *
 *     Annuity
 *     Lease (with buy back or residual)*
 *     Loan or Mortgage (with balloon)*
 *
```

**Pull request closed without merging**